package.json

```diff
@@ -57,10 +61,6 @@
   },
   "proxy": "http://localhost:8081",
   "devDependencies": {
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb": "^19.0.0",
-    "eslint-plugin-react": "^7.27.0",
-    "eslint-plugin-testing-library": "^5.0.0",
```

Comment:
We need these dependencies for linting and testing. This change should be reverted.

Reply:
This was my attempt to fix my npm run build issues. I had to use another IDE to see any changes and I was looking up how to resolve these issues. I was able to fix the npm run build issue and I will put these dependencies back.